### PR TITLE
feat: Added product and order page

### PR DESCRIPTION
**Pull request by mithmusq3**
Two different Product Models have been taken while rebasing (should resolve),
Creating this Pr to build and present

### CODE REVIEW

File: lib/views/order_page.dart [hunk 1]
```diff
@@ -0,0 +1,217 @@
...
+
+class OrderPage extends GetView<OrderController> {
+  const OrderPage({Key? key}) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    final ordersController = Get.find<OrderController>();
+    final ordersListMain = Get.arguments;
+    if (ordersController.flag.value) {
+      ordersController.copy(ordersListMain);
```
Comment (Ram-20062003) [inline lib/views/order_page.dart:15]: use of copy here?

**Review by Ram-20062003 — CHANGES_REQUESTED** [review]

**Comment by Ram-20062003** [comment]
Pull rebase your PR

**Comment by mithmusq3** [comment]
Requested Changes has been addressed

File: lib/views/product_page.dart [hunk 2]
```diff
@@ -0,0 +1,178 @@
...
+          child: Column(
+            children: [
+              SingleChildScrollView(
+                child: Obx(
+                  () => SizedBox(
+                    height: MediaQuery.of(context).size.height - 60,
+                    child: ListView.builder(
+                      itemBuilder: (ctx, index) => Column(children: [
+                        Container(
+                          height: 180 >
```
Comment (Ram-20062003) [inline lib/views/product_page.dart:26]: this if conditon is not needed ig

**Review by Ram-20062003 — CHANGES_REQUESTED** [review]

**Pull request merged**